setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 setup(
     name='orchestra',
-    version='0.1',
+    version='0.1.2',
```

Comment:
@thisisdhaas is writing instructions on how to do this properly, but at a high level, we shouldn't change versions in PRs that change functionality.  we'll change the version in a subsequent PR, which might even get automated.